src/attendee/api/views.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+from rest_framework import views, status
+from rest_framework.response import Response
+
+from registry.helper import reg
+from ext2020.models import Attendee
+
+
+class AttendeeAPIView(views.APIView):
+    model = Attendee
+
+    def post(self, request):
+        self.token = request.data['token']
+        try:
+            Attendee.objects.get(token=self.token)
+            slug = settings.CONFERENCE_DEFAULT_SLUG
+            key_filter = f"{slug}.live"
+            num_rooms = [key for key in reg if key_filter in str(key)]
+
+            response_data = {"youtube_infos": []}
+            for idx, room in enumerate(num_rooms):
+                response_data["youtube_infos"].append({
+                    "room": f"R{idx+1}",
+                    "video_id": reg.get(f'{room}', '')
+                })
```

Comment:
Let's take the last part of the registry key as the room name. The following suggestion is not tested:

```suggestion
            for idx, room_key in enumerate(reg_room_keys):
                pattern = r'{slug}\.live\.(?P<room_name>[0-9a-zA-Z]*)'
                match_result = re.match(pattern, room_key)
                if not match_result:
                    continue
                room_name = match_result.group(1)
                
                response_data["rooms"].append({
                    "room": room_name,
                    "video_id": reg.get(room_key, '')
                })
```